streamwebs_frontend/streamwebs/templates/streamwebs/create_site.html

```diff
@@ -0,0 +1,33 @@
+{% extends 'streamwebs/base.html' %}
+{% load i18n %}
+{% load staticfiles %}
+
+{% block scripts %}
+<script src="http://maps.google.com/maps/api/js?key=AIzaSyBGA5zk8i-l6YnlWaLedx0TXhAWNibN2hE&callback=initMap" async defer></script>
```

Comment:
Either call initMap() in the jQuery ready callback in create_site.js or give it as a callback in the Google Maps request, but don't do both.